NitroxClient/MonoBehaviours/EntityPositionBroadcaster.cs

```diff
@@ -62,7 +62,14 @@ private List<EntityTransformUpdate> BuildUpdates(Dictionary<NitroxId, GameObject
         {
             if (gameObjectWithId.Value)
             {
-                updates.Add(new RawTransformUpdate(gameObjectWithId.Key, gameObjectWithId.Value.transform.position.ToDto(), gameObjectWithId.Value.transform.rotation.ToDto()));
+                if (ShouldBroadcastLocalPosition(gameObjectWithId.Value))
+                {
+                    updates.Add(new RawTransformUpdate(gameObjectWithId.Key, gameObjectWithId.Value.transform.localPosition.ToDto(), gameObjectWithId.Value.transform.localRotation.ToDto()));
```

Comment:
As far as I can see this new behavior isn't reflected on client or server. What is the reason of this?